azkaban-common/src/test/java/azkaban/ServiceProviderTest.java

```diff
@@ -44,9 +42,10 @@
   // Test if one class is singletonly guiced. could be called by
   // AZ Common, Web, or Exec Modules.
   public static void assertSingleton(final Class azkabanClass, final Injector injector) {
-    final Object azkabanObj1 = requireNonNull(injector.getInstance(azkabanClass));
-    final Object azkabanObj2 = requireNonNull(injector.getInstance(azkabanClass));
-    assertTrue(azkabanObj1 == azkabanObj2);
+    assertThat(injector.getInstance(azkabanClass)).isNotNull();
+    final Object azkabanObj1 = injector.getInstance(azkabanClass);
+    final Object azkabanObj2 = injector.getInstance(azkabanClass);
+    assertThat(azkabanObj1 == azkabanObj2).isTrue();
```

Comment:
Is http://joel-costigliola.github.io/assertj/core/api/org/assertj/core/api/AbstractAssert.html#isSameAs(java.lang.Object) better?